### PR TITLE
Updating README for katello-cli

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+If you like living on the edge, go for
+[nightly builds](https://fedorahosted.org/katello/wiki/InstallTesting)
+instead.
+
+Found a bug?
+------------
+
+That's rather unfortunate. But don't worry! We can help. Just
+[file a bug](https://bugzilla.redhat.com/enter_bug.cgi?product=Katello).
```
Comment (bbuckingham) [inline README.md]: Should we mention that providing feedback via github issue is also an option?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+From the virtualenv:
+
+```
+env SWIG_FEATURES="-cpperraswarn -includeall -D__`uname -m`__ -I/usr/include/openssl" pip install -r <env-name>/etc/katello-cli/requirements.txt
+```
+
+Contact & Resources
+-------------------
+
```
Comment (bbuckingham) [inline README.md:73]: Might also want to include http://katello.org as a resource.

**Comment by bbuckingham** [comment]
Good start.  ACK

**Comment by daviddavis** [comment]
@bbuckingham I made the same changes in https://github.com/Katello/katello/pull/2161 if you'd like to ack that as well. Thanks.

**Pull request merged**